doc/source/whatsnew/v0.24.0.txt

```diff
@@ -726,6 +748,18 @@ Reshaping
 - Bug in :meth:`Series.replace` and meth:`DataFrame.replace` when dict is used as the `to_replace` value and one key in the dict is is another key's value, the results were inconsistent between using integer key and using string key (:issue:`20656`)
 - Bug in :meth:`DataFrame.drop_duplicates` for empty ``DataFrame`` which incorrectly raises an error (:issue:`20516`)
 
+.. _whatsnew_0240.bug_fixes.sparse:
+
+Sparse
+^^^^^^
+
+- Updating a boolean, datetime, or timedelta column to be Sparse now works (:issue:`22367`)
+- Bug in :meth:`Series.to_sparse` with Series already holding sparse data not constructing properly (:issue:`22389`)
+- Providing a ``sparse_index`` to the SparseArray constructor no longer defaults the na-value to ``np.nan`` for all dtypes. The correct na_value for ``data.dtype`` is now used.
+- Bug in ``SparseArray.nbytes`` under-reporting its memory usage by not including the size of its sparse index.
+- Improved performance of :meth:`Series.shift` for non-NA ``fill_value``, as values are no longer converted to a dense array.
+- A SparseDtype with boolean subtype is considered bool by :meth:`api.types.is_bool_dtype`.
```

Comment:
issue for this last one?

Also might be ok to just put these in the sparse changes section of whatsnew

Reply:
https://github.com/pandas-dev/pandas/issues/22326 is the general issue. Basically  just need to

1. determine how we want to treat EAs with a bool dtype
2. switch my `is_sparse(arr)` to `is_extension_array_dtype(arr)
3. document what that changes

Reply:
ok, even ok to put this PR issue number there, just so they can be clicked by a user

Reply:
Actually, going to remove this entry since it's covered under the note from #22326.